3-tutorial.md

```diff
@@ -622,7 +622,7 @@ You may notice the *Endpoint*, just like *Deployments* create *ReplicaSets* whic
 will differ).
 
 ```bash
-❯ kubectl get endpoint kuard -o yaml
+❯ kubectl get endpoints kuard -o yaml
```

Comment:
can you change to `endpoints/kuard` since that is the syntax used in the rest of the guide